test/powershell/Modules/Microsoft.PowerShell.Utility/Export-FormatData.Tests.ps1

```diff
@@ -14,33 +21,70 @@ Describe "Export-FormatData DRT Unit Tests" -Tags "CI" {
             $content.Contains($formatViewDefinition[$i].Name) | Should -BeTrue
         }
     }
-}
-
-Describe "Export-FormatData" -Tags "CI" {
-
-    $testOutput = Join-Path -Path $TestDrive -ChildPath "outputfile"
-
-    AfterEach {
-        Remove-Item $testOutput -Force -ErrorAction SilentlyContinue
-    }
 
-    Context "Check Export-FormatData can be called validly." {
-	    It "Should be able to be called without error" {
-	        { Get-FormatData | Export-FormatData -Path $testOutput } | Should -Not -Throw
-	    }
+    It "Should have a valid xml tag at the start of the file" {
+        $fd | Export-FormatData -Path $testOutput
+        $piped = Get-Content $testOutput -Raw
+        $piped[0] | Should -BeExactly "<"
     }
 
-    Context "Check that the output is in the correct format" {
-	    It "Should not return an empty xml file" {
-	        Get-FormatData | Export-FormatData -Path $testOutput
-	        $piped = Get-Content $testOutput
-	        $piped | Should -Not -BeNullOrEmpty
-	    }
+    It "Should well format output xml" {
+        $xmlContent=@"
+            <Configuration>
+            <ViewDefinitions>
+            <View>
+            <Name>ExportFormatDataName</Name>
+            <ViewSelectedBy>
+                <TypeName>ExportFormatDataTypeName</TypeName>
+            </ViewSelectedBy>
+            <TableControl>
+                <TableHeaders />
+                <TableRowEntries>
+                <TableRowEntry>
+                <TableColumnItems>
+                <TableColumnItem>
+                    <PropertyName>Guid</PropertyName>
+                </TableColumnItem>
+                </TableColumnItems>
+                </TableRowEntry>
+                </TableRowEntries>
+            </TableControl>
+            </View>
+            </ViewDefinitions>
+            </Configuration>
+"@
+        $expected = @"
+<?xml version="1.0" encoding="utf-8"?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>ExportFormatDataName</Name>
+      <ViewSelectedBy>
+        <TypeName>ExportFormatDataTypeName</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableHeaders />
+        <TableRowEntries>
+          <TableRowEntry>
+            <TableColumnItems>
+              <TableColumnItem>
+                <PropertyName>Guid</PropertyName>
+              </TableColumnItem>
+            </TableColumnItems>
+          </TableRowEntry>
+        </TableRowEntries>
+      </TableControl>
+    </View>
+  </ViewDefinitions>
+</Configuration>
+"@
+        $testfilename = [guid]::NewGuid().ToString('N')
```

Comment:
Would be nice if we could just use `New-TemporaryFile`, maybe we need to add a `-Path` parameter so we can pass in $TestDrive.  I'm fine leaving this as-is.